docs/en/install_deploy.md

```diff
@@ -0,0 +1,393 @@
+
+* Modify `endpoint`. The endpoint is the deployment machine ip/domain name and port number separated by colons.
+* Modify `nameserver` to be the address of Nameserver.
+
+```
+--endpoint=172.27.128.33:8080
+--nameserver=172.27.128.33:6527
+```
+
+**Notice:**
+
+* The endpoint cannot use 0.0.0.0 and 127.0.0.1. You can also choose not to set `--endpoint`, and only configure the port number `--port`.
+
+#### 3 Start the service
+
+```
+sh bin/start.sh start standalone_apiserver
+```
+
+## Deploy Cluster Version
+
+OpenMLDB cluster version needs to deploy Zookeeper, Nameserver, Tablet and other modules. Among them, Zookeeper is used for service discovery and saving metadata information. The Nameserver is used to manage the tablet, achieve high availability and failover. Tablets are used to store data and synchronize data between master and slave. APIServer is optional. If you want to interact with OpenMLDB in http, you need to deploy this module.
+
+**Notice:** It is best to deploy different components in different directories for easy upgrades individually. If multiple tablets are deployed on the same machine, they also need to be deployed in different directories.
+
+### Deploy Zookeeper
+
+It is recommended to deploy version 3.4.14. If there is an available zookeeper cluster, you can skip this step.
+
+#### 1. Download the zookeeper installation package
+
+```
+wget https://archive.apache.org/dist/zookeeper/zookeeper-3.4.14/zookeeper-3.4.14.tar.gz
+cd zookeeper-3.4.14
+cp conf/zoo_sample.cfg conf/zoo.cfg
+```
+
+#### 2. Modify the Configuration File
+
+Open the file `conf/zoo.cfg` and modify `dataDir` and `clientPort`.
+
+```
+dataDir=./data
+clientPort=7181
+```
+
+#### 3. Start Zookeeper
+
+```
+sh bin/zkServer.sh start
+```
+
+Deploy the Zookeeper cluster [refer to here](https://zookeeper.apache.org/doc/r3.4.14/zookeeperStarted.html#sc_RunningReplicatedZooKeeper).
+
+### Deploy tablet
+
+#### 1 Download the OpenMLDB deployment package
+
+```
+wget https://github.com/4paradigm/OpenMLDB/releases/download/v0.4.3/openmldb-0.4.3-linux.tar.gz
+tar -zxvf openmldb-0.4.3-linux.tar.gz
+mv openmldb-0.4.3-linux openmldb-tablet-0.4.3
+cd openmldb-tablet-0.4.3
+```
+
+#### 2 Modify the Configuration File: conf/tablet.flags
+
+* Modify `endpoint`. The endpoint is the deployment machine ip/domain name and port number separated by colons.
+* Modify `zk_cluster` to the already started zk cluster address.
+* If you share zk with other OpenMLDB, you need to modify `zk_root_path`.
+
+```
+--endpoint=172.27.128.33:9527
+--role=tablet
+
+# If tablet run as cluster mode zk_cluster and zk_root_path should be set:
+--zk_cluster=172.27.128.33:7181,172.27.128.32:7181,172.27.128.31:7181
+--zk_root_path=/openmldb_cluster
+```
+
+**Notice:**
+* The endpoint cannot use 0.0.0.0 and 127.0.0.1.
+* If the domain name is used here, all the machines where the client using openmldb is located must be equipped with the corresponding host. Otherwise, it will not be accessible.
```

Comment:
all the machines where the client using openmldb is located must be equipped with the corresponding host.
=>
all the machines with OpenMLDB clients must be configured with the corresponding host.

Reply:
Ok, thank you.